src/main/fc/rc.c

```diff
@@ -689,15 +689,19 @@ FAST_CODE_NOINLINE void updateRcCommands(void)
 
         rcCommandBuff.X = rcCommand[ROLL];
         rcCommandBuff.Y = rcCommand[PITCH];
-        if ((!FLIGHT_MODE(ANGLE_MODE) && (!FLIGHT_MODE(HORIZON_MODE)) && (!FLIGHT_MODE(GPS_RESCUE_MODE)))) {
+        if ((!FLIGHT_MODE(ANGLE_MODE) && !FLIGHT_MODE(ALTHOLD_MODE)
```

Comment:
Please, try to be consistent with other places (multiple tests vs bitmask)

Reply:
Fixed.